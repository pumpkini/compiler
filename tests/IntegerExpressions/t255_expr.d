int main() {
    int a;
    int number;
    int b;
    int c;
    int d;
    d=2;
    a=number=85*(b=2*3)*(c=d);
    Print(d);
    Print(c);
    Print(b);
    Print(number);
    Print(a);
}