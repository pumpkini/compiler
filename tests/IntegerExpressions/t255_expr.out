2
2
6
1020
1020
